src/test/java/bio/terra/controller/StudyConnectedTest.java

```diff
@@ -0,0 +1,64 @@
+package bio.terra.controller;
+
+import bio.terra.category.Connected;
+import bio.terra.fixtures.JsonLoader;
+import bio.terra.fixtures.Names;
+import bio.terra.model.DeleteResponseModel;
+import bio.terra.model.StudyRequestModel;
+import bio.terra.model.StudySummaryModel;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+@Category(Connected.class)
+public class StudyConnectedTest {
+    @Autowired private MockMvc mvc;
+    @Autowired private ObjectMapper objectMapper;
+    @Autowired private JsonLoader jsonLoader;
+
+    @Test
+    public void testCreateOmopStudy() throws Exception {
+        StudyRequestModel studyRequest = jsonLoader.loadObject("it-study-omop.json", StudyRequestModel.class);
+        studyRequest.name(Names.randomizeName(studyRequest.getName()));
+
+        MvcResult result = mvc.perform(post("/api/repository/v1/studies")
+            .contentType(MediaType.APPLICATION_JSON)
+            .content(objectMapper.writeValueAsString(studyRequest)))
+            .andExpect(status().isCreated())
+            .andReturn();
+        MockHttpServletResponse response = result.getResponse();
+        assertThat("create omop study successfully", response.getStatus(), equalTo(HttpStatus.CREATED.value()));
+        StudySummaryModel studySummaryModel =
+            objectMapper.readValue(response.getContentAsString(), StudySummaryModel.class);
+
```

Comment:
Worth checking that the study exists using the PDAO?

Reply:
I'm not sure I should keep the test at all. I almost deleted it, but left it in the PR to get other opinions. It is redundant with the integration test. If we keep it, then yes, we could have it do other things.